Moco/Moco/MocoGoal/MocoContactTrackingGoal.h

```diff
@@ -69,13 +70,13 @@ class OSIMMOCO_API MocoContactTrackingGoalGroup : public Object {
 /// - \f$ \mathrm{proj}_{\hat{n}}() \f$: this function projects the force error
 ///     either onto \f$ \hat{n} \f$ or onto the plane perpendicular to
 ///     \f$ \hat{n} \f$.
-/// - \f$ F_{m,g} \f$ the sum of the contact forces in group \f$ g \f$,
+/// - \f$ \vec{F}_{m,j} \f$ the sum of the contact forces in group \f$ j \f$,
 ///     expressed in ground.
-/// - \f$ F_{e,g} \f$ the experimental contact force for group \f$ g \f$,
+/// - \f$ \vec{F}_{e,j} \f$ the experimental contact force for group \f$ j \f$,
 ///     expressed in ground.
 ///
-/// If the model's gravity $g$ is 0, then we normalize by the total mass of the
-/// system instead of the total weight.
+/// If the model's gravity \f$ g \f$ is 0, then we normalize by the total mass
+/// of the system instead of the total weight.
```

Comment:
Again not part of the PR (and could be ignored or deferred to another PR), but this statement was a little confusing to me with "instead of" because the equation above does not show any normalization.

Reply:
Addressed.